src/sentry/utils/snuba.py

```diff
@@ -80,6 +80,11 @@
 }
 
 
+def parse_snuba_datetime(value):
+    """Parses a datetime value from snuba."""
+    return parse_datetime(value)
```

Comment:
Why do we need this function?

Reply:
I added the function because `parse_datetime` tries a bunch of options. This will let us swap this out for one that is more optimized and does not try 4 different variants later. It also does not provide any other parameters that might accidentally be parsed.